test/e2ekafka/knative_kafka_test.go

```diff
@@ -78,7 +78,7 @@ func TestKnativeKafka(t *testing.T) {
 	}
 
 	t.Cleanup(func() {
-		_ = caCtx.Clients.Kafka.MessagingV1beta1().KafkaChannels(knativeKafkaNamespace).Delete(context.Background(), "", metav1.DeleteOptions{})
+		_ = caCtx.Clients.Kafka.MessagingV1beta1().KafkaChannels(knativeKafkaNamespace).Delete(context.Background(), ch.Name, metav1.DeleteOptions{})
```

Comment:
Btw we might need to avoid this due to this: https://github.com/openshift-knative/serverless-operator/pull/1010/files

Reply:
not sure, yet ...
see also: https://github.com/openshift-knative/serverless-operator/pull/1010

but thanks for the pointer!